### DISCUSSION TITLE
tmpdir_stagger_phase_major vs tmpdir_stagger_phase_minor

### DISCUSSION THREAD

**Discussion by windkostya** [ericaltendorf/plotman#589] [Q&A] [answered]
Can someone please explain me like i'm 2 years old these 4 parameters. I do apologize right away. 

tmpdir_stagger_phase_major  -- Does this mean the next plot will start in stage 3 if i have here 3?  
tmpdir_stagger_phase_minor  -- what his mean? 
tmpdir_stagger_phase_limit    -- what his mean?
global_stagger_m -- based on my understanding that is when the next plot will start. but this setting will be overwritten by the tmpdir_stagger_phase_major ? correct?

Please help me to understand this configuration.

Thank you,
Cons
![config](https://user-images.githubusercontent.com/17420563/118868821-82450300-b899-11eb-8907-bd82a6921935.png)

**Comment by mruckli** [comment, marked as answer]
I would suggest you to check out this reply and see id it is still unclear afterwards: https://github.com/ericaltendorf/plotman/issues/151#issuecomment-832980275

**Reply by windkostya** [reply]
Awesome. This is now make sense. Thank you so much for sharing the link.
